### ISSUE TITLE
cannot import name Reminder

### ISSUE THREAD

**Issue by rojobuffalo** [RasaHQ/rasa_core#349]
**Rasa Core version**: 0.8.5

**Python version**: 3.6.5

**Operating system** (windows, osx, ...): OSX

**Issue**: cannot import Reminder

Error output:
```
  File "/Users/myusername/Code/mybot/actions.py", line 7, in <module>
    from rasa_core.events import Reminder, SlotSet
ImportError: cannot import name 'Reminder'
```

I installed rasa_core with `pip install rasa_core`. Bot works fine until I try to add this Reminder functionality.

**Comment by amn41** [comment]
should be `ReminderScheduled`  
https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/events/__init__.py#L455 

I can see that the docs are out of date - would you care to create a PR to fix that?

**Comment by rojobuffalo** [comment]
Sure, is it just s/Reminder/ReminderScheduled ?

**Comment by akelad** [comment]
yes

**Comment by tmbo** [comment]
This has been fixed in #357
